pkg/services/ngalert/notifier/channels/line.go

```diff
@@ -18,7 +18,7 @@ import (
 	old_notifiers "github.com/grafana/grafana/pkg/services/alerting/notifiers"
 )
 
-const (
+var (
```

Comment:
Is this necessary? I don't think that this value ever changes.

Reply:
`channels.LineNotifyURL` is actually being changed in the integration test